src/java/org/apache/cassandra/service/StorageService.java

```diff
@@ -4525,16 +4525,39 @@ public List<String> getParentRepairStatus(int cmd)
                ImmutableList.<String>builder().add(pair.left.name()).addAll(pair.right).build();
     }
 
+    @Deprecated
+    @Override
     public void setRepairSessionMaxTreeDepth(int depth)
     {
         DatabaseDescriptor.setRepairSessionMaxTreeDepth(depth);
     }
 
+    @Deprecated
+    @Override
     public int getRepairSessionMaxTreeDepth()
     {
         return DatabaseDescriptor.getRepairSessionMaxTreeDepth();
     }
 
+    @Override
+    public void setRepairSessionMaximumTreeDepth(int depth)
+    {
+        try
+        {
+            DatabaseDescriptor.setRepairSessionMaxTreeDepth(depth);
+        }
+        catch (ConfigurationException e)
+        {
+            throw new IllegalArgumentException(e.getMessage());
+        }
+    }
+
+    @Override
+    public int getRepairSessionMaximumTreeDepth()
```

Comment:
can you add a comment explaining why this exists?  Fo someone reviewing the code its going to be asked "why is getRepairSessionMaxTreeDepth deprecated... its the same logic here...", so if we are going to depreciate the `get` we should explain why in the code

Reply:
Good point, we can also add the ticket number for clarity